AppStore/Ukrainian/Description.iOS.txt

```diff
@@ -1,28 +1,28 @@
-Чи цікавились ви колись «хто ця актриса?», або можливо «коли вийшов цей фільм?», або «як називався той епізод мого улюбленого серіалу»? Може, ви знавець кінематографу і замислювались «скільки їм було коли вийшов цей фільм»?
+Чи цікавились ви колись «хто ця актриса?», або можливо «коли вийшов цей фільм?», або «як називався той епізод мого улюбленого серіалу»? Може, ви знавець кінематографу і замислювались «коли вийшов цей фільм, скільки років було режисеру»?
 
-Чи хотіли б ви дізнатися відповіді на усі ці питання без постійних запитів створити обліковий запис? Без реклами та без відео, що автоматично починають грати? Не покладаючись на міжнародний конгломерат, який намагається вам щось продати?
+Чи хотіли б ви дізнатися відповіді на усі ці питання без постійних запитів створити обліковий запис? Без відео, що автоматично починають грати? Не покладаючись на міжнародний конгломерат, який усюди пхає свою рекламу?
 
 Тоді вам сподобається Callsheet.
 
-Callsheet — це найкращий спосіб дізнатися інформацію про акторів та знімальну групу фільмів та серіалів. Це програма яка поважає ваш час, не намагається продати вам непотрібні речі, і зроблена щоб швидко та просто відповісти на ваші запитання.
+Callsheet — це найкраща програма для того, щоб дізнатися інформацію про акторів і знімальну групу фільмів та серіалів. Вона поважає ваш час, не намагається продати вам непотрібні речі, і швидко та легко відповідає на ваші запитання.
 
-Callsheet також має деякі переваги, яких не було у вашій минулій улюбленій програмі-довіднику:
+Деякі переваги Callsheet, яких не було у вашій попередній програмі-довіднику:
 
 • Уникайте спойлерів у серіалах, ховаючи імена персонажів, кількість епізодів, їх назву, або кадри з них.
   Сховайте ці деталі, щоб зарано не дізнатися таємну особистість персонажів або що вони скоро покинуть серіал.
 • Швидкий доступ до цікавих фактів, Вікіпедії, інформації щодо батьківського нагляду, та сервісів для перегляду фільму або серіалу (надано сервісом JustWatch).
-• Закріплюйте для швидкого доступу серіали, фільми, акторів, або знімальну групу.
+• Закріплюйте серіали, фільми, акторів, або знімальну групу для швидкого доступу.
 • Знаходьте інформацію яку ви вже шукали, завдяки історії пошуків.
 • Легко дізнайтеся скільки було людям, що працювали над фільмом, *коли той фільм вийшов* 
 • Легко дізнайтеся вік людини для кожного запису її фільмографії
 • Легко дізнайтеся зріст акторів (коли можливо)
 • Підтримка темного режиму
 • Підтримка VoiceOver
 • Альтернативні іконки для підписників
-• Автоматична та анонимна інтеграція з програмою Channels
-• Експериментальна та анонимна інтеграція з Plex
+• Автоматична інтеграція з програмою Channels
+• Експериментальна інтеграція з Plex
```

Comment:
Couldn't think of a short and accurate translation for "no-login"

"anonymous integration" felt weird, so I've removed "no-login" altogether